src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -186,11 +199,18 @@ export default {
       type: String,
       default: 'No results found',
     },
+    isMultiple: {
+      type: Boolean,
+      default: false,
+    },
   },
   data() {
     return {
       isOpen: false,
       filterText: '',
+      visibleWindow: null,
+      active: null,
```

Comment:
I'd name it lastHighlightedItem and set it in `select()` method regardless whether it is multiple or single. That means you don't need currentActiveItem computed property